data_exploration/unified_metadata_schema/unified_metadata_schema.py

```diff
@@ -0,0 +1,300 @@
+import json
```

Comment:
This Python code should follow the Black code style to fit in with the rest of the code base. To do that, can you please rebase the branch against `main` (because it contains the relevant config), install pre-commit, and run `pre-commit run --all-files`

Reply:
(Also, when you set it up, it will automatically check & re-format all changes you make to follow the Black code style going forward)